README.md

```diff
@@ -1,3 +1,21 @@
+<!--- BADGES: START --->
+[![GitHub - License](https://img.shields.io/github/license/koaning/doubtlab?logo=github&style=flat&color=green)][#github-license]
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/doubtlab?logo=pypi&style=flat&color=blue)][#pypi-package]
+[![PyPI - Package Version](https://img.shields.io/pypi/v/doubtlab?logo=pypi&style=flat&color=orange)][#pypi-package]
+[![Conda - Platform](https://img.shields.io/conda/pn/conda-forge/doubtlab?logo=anaconda&style=flat)][#conda-forge-package]
```

Comment:
What does the conda platform indicate exactly?

Reply:
When a package is made available on a conda channel, you could make it available selectively for any or a combination of operating systems: windows, mac os, linux/unix. Typically, if the library does not have some os-specific limitations, the preferred way of packaging it on conda-forge channel is to make it architecture agnostic (`noarch`).

The conda `platform` basically specifies what platforms (os: win/mac/unix) it is available for. In our case, it is `noarch` (it is architecture-agnostic).

![image](https://user-images.githubusercontent.com/10201242/144410173-94f16f5a-c383-4802-abbf-81de300e4c8a.png)

For instance, `boto3` library is not packaged as `noarch` ALONE, and it is possible to have different versions of the same package available for different platforms.

![image](https://user-images.githubusercontent.com/10201242/144410714-1b99e65b-15bc-4a86-b7a6-8e9f1e1033e7.png)